.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   cli:
     docker:
-      - image: circleci/circleci-cli
+      - image: circleci/circleci-cli:0.1.25725
```

Comment:
Lets pin the image tag to avoid pulling in potential breaking changes via unspecified `latest` tag.